### PR TITLE
Added title of address

### PR DESCRIPTION
**Pull request by wachterjohannes**

### CODE REVIEW

**Comment by turbo-ele** [comment]
ack

**Comment by danrot** [comment]
No ACK on PRs with failing checks :stuck_out_tongue: And what kind of title has an address?

**Comment by wachterjohannes** [comment]
@danrot forget @turbo_ele 's ack i have only bootstraped an additional field for the address! but ele has to talk to bernd and oli how and if this will be in standard!

**Comment by danrot** [comment]
One of my questions remain: What is the title of an address?

**Comment by wachterjohannes** [comment]
its like a simple "title" like "my home address" or "my liitle ponyhof" (: to organize multiple addresses for the profile!

File: src/Sulu/Bundle/ContactBundle/Resources/public/js/services/contact-manager.js [hunk 1]
```diff
@@ -64,7 +64,7 @@ define([
             if (!!data.categories) {
                 contact.get('categories').reset();
                 Util.foreach(data.categories, function(id) {
-                    var category = Category.findOrCreate({id: id});
+                    var category = Category.findOrCreate({id: (!id.id ? id : id.id)});
```
Comment (danrot) [inline src/Sulu/Bundle/ContactBundle/Resources/public/js/services/contact-manager.js]: @wachterjohannes I am going to revert this change. Doesn't look like this change has a consequence on this feature, and it looks to me like there is a wrong call else where, and this issue should be fixed there, not here.
